client/components/Nav/NavBar.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react'
```

Comment:
Can you squash your commits into 1 commit?

Please attach a screenshot of the header component. It crashes on Netlify, hence I'm unable to view it